source/extensions/stat_sinks/metrics_service/grpc_metrics_proto_descriptors.cc

```diff
@@ -10,6 +12,8 @@ namespace StatSinks {
 namespace MetricsService {
 
 void validateProtoDescriptors() {
+  const envoy::service::metrics::v2::StreamMetricsMessage _dummy_v2;
```

Comment:
Can this be in protobuf_link_hacks? That way we can centralize this and more easily cleanup later.

Reply:
I don't know all the intricacies of the extensions build system but I thought that because this was an extension I would leave this under the extensions tree. I'll put a link to the tracking issue once I create it.

Reply:
SGTM